packages/component-library/draft/Kaizen/GuidanceBlock/GuidanceBlock.tsx

```diff
@@ -0,0 +1,128 @@
+import { Button, Icon, Text } from "@kaizen/component-library"
+const configureIcon = require("@kaizen/component-library/icons/arrow-forward.icon.svg")
+  .default
+
+import classnames from "classnames"
+import * as React from "react"
+
+const styles = require("./GuidanceBlock.scss")
+
+const closeIcon = require("@kaizen/component-library/icons/close.icon.svg")
+  .default
+
+type Props = {
+  img: {
+    src: string
+    alt: string
+  }
+  text: {
+    title: string
+    description: string | React.ReactNode
+  }
+  button: {
+    label: string
+    onClick: () => void
+  }
+  onDismiss: () => void
+}
+
+type State = {
+  hidden: boolean
+  removed: boolean
+}
+class GuidanceBlock extends React.Component<Props, State> {
+  state = {
+    hidden: false,
+    removed: false,
+  }
+
+  containerRef = React.createRef<HTMLDivElement>()
+
+  constructor(props: Props) {
+    super(props)
+
+    this.dismissBanner = this.dismissBanner.bind(this)
+    this.onTransitionEnd = this.onTransitionEnd.bind(this)
+  }
+
+  dismissBanner(): void {
+    this.setState({ hidden: true })
+    this.props.onDismiss()
+  }
+
+  onTransitionEnd(e: React.TransitionEvent<HTMLDivElement>) {
+    // Be careful: this assumes the final CSS property to be animated is "margin-top".
+    if (this.state.hidden && e.propertyName === "margin-top") {
+      this.setState({ removed: true })
+    }
+  }
+
+  render(): JSX.Element | null {
+    if (this.state.removed) {
+      return null
+    }
+
+    return (
+      <div
+        className={this.bannerClassName()}
+        style={{ marginTop: this.marginTop() }}
+        ref={this.containerRef}
+        onTransitionEnd={this.onTransitionEnd}
+      >
+        <div>
+          <img
+            src={this.props.img.src}
+            alt={this.props.img.alt}
+            height="155px"
+            width="155px"
+          />
+        </div>
+        <div className={styles.descriptionContainer}>
+          <div className={styles.headingWrapper}>
+            <Text inline={true} tag="h3">
+              {this.props.text.title}
+            </Text>
+          </div>
+          <Text tag="p">{this.props.text.description}</Text>
```

Comment:
```suggestion
          <Text tag="p" style="body">{this.props.text.description}</Text>
```
Same as above. I went and checked the styles for the Text component to see what style the `p` tag falls back to, and it's `body`.